src/services/batch_service.js

```diff
@@ -25,18 +30,19 @@ export default (easypostClient) =>
     }
 
     /**
-     * Add shipments to a batch.
-     * @param {number} id
-     * @param {Array} shipmentIds
-     * @returns {this}
+     * Add {@link Shipment shipments} to a {@link Batch batch}.
+     * See {@link https://www.easypost.com/docs/api/node#add-shipments-to-a-batch EasyPost API Documentation} for more information.
+     * @param {number} id - The id of the batch to add shipments to.
```

Comment:
```suggestion
     * @param {string} id - The id of the batch to add shipments to.
```

Please replace `number` with `string` for the `id` parameters. This apples to all occurrences in `batch_service`